tests/unittests/CoreText/CTLineTests.mm

```diff
@@ -27,7 +27,7 @@
     [string addAttribute:NSForegroundColorAttributeName value:[UIColor redColor] range:wholeRange];
     [string addAttribute:NSFontAttributeName value:font range:wholeRange];
 
-    return string;
+    return [string autorelease];
```

Comment:
whoops, I think these were remnants from switching between branches a bunch.

---
In reply to: [76315512](https://github.com/Microsoft/WinObjC/pull/909#discussion_r76315512) [](ancestors = 76315512)